pom.xml

```diff
@@ -95,6 +95,11 @@
             <artifactId>json</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
would be better to change the implementation to read version without additional dependency. Community extension users will have to download this jar manually, in addition to bson, driver-core and driver-sync